patcher_arm.go

```diff
@@ -0,0 +1,8 @@
+// +build arm
```

Comment:
Now `patcher_arm.go` and `patcher_arm64.go` are basically the same, so I would transform this file to `patcher_unsupported.go` and write the following build flags:

```suggestion
// +build !386
// +build !amd64
```

This should compile if the architecture is not x86 neither x64. Please test it :)